packages/component-model-semantic-extraction/entities/semanticExtraction/index.test.js

```diff
@@ -36,8 +36,28 @@ describe('SemanticExtraction', () => {
     it('return an entity with title filedName when called', () => {
       const titleExtraction = SemanticExtraction.createTitleEntity(
         'manuscriptId',
+        'test title',
       )
       expect(titleExtraction.fieldName).toBe('title')
+      expect(titleExtraction.value).toBe('test title')
+    })
+  })
+
+  describe('findByManuscriptId', () => {
```

Comment:
Does this test call the database? I was thinking it might be a good idea to mark tests that do this with some kind of annotation moving forward, so when the time comes to decouple we can find them easily.

Reply:
Good idea, alternatively the tests can be re-written with a mocked database. If they require the actual database then either we are testing Objection/Knex at this point and the test can probably be removed or it means we have too much logic in our ORM model

Reply:
Yeah we are stuck in this place at the moment ... would love to chat about this sometime - maybe next retro?